exporter.go

```diff
@@ -385,6 +387,48 @@ func extractVal(s string) (val float64, err error) {
 	return
 }
 
+/*
+	Valid Examples
+	id=11 addr=127.0.0.1:63508 fd=8 name= age=6321 idle=6320 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=setex
+	id=14 addr=127.0.0.1:64958 fd=9 name= age=5 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=26 qbuf-free=32742 obl=0 oll=0 omem=0 events=r cmd=client
+*/
+func parseClientListString(clientInfo string) (id float64, host string, port string, name string, age string, idle string, flags string, db string, cmd string, ok bool) {
+	var err error
+	ok = false
+	if matched, _ := regexp.MatchString(`^id=\d+ addr=\d+`, clientInfo); !matched {
+		return
+	}
+	connectedClient := map[string]string{}
+	for _, kvPart := range strings.Split(clientInfo, " ") {
+		vPart := strings.Split(kvPart, "=")
+		if len(vPart) != 2 {
+			log.Debugf("Invalid format for client list string, got: %s", kvPart)
+			return
+		}
+		connectedClient[vPart[0]] = vPart[1]
+	}
+
+	if id, err = strconv.ParseFloat(connectedClient["id"], 64); err != nil {
```

Comment:
`id` is no longer used, you can remove this

Reply:
id is used
check extractConnectedClients function